dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=qBittorrent Daemon Service for user %I
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox --webui-port=8080
```

Comment:
Since web UI port can be set via config file (and as such via web UI itself), this breaks the POLA principle.